packages/web-components/src/components/va-process-list/va-process-list.scss

```diff
@@ -0,0 +1,90 @@
+@use 'uswds-helpers/src/styles/usa-sr-only';
+@use 'usa-process-list/src/styles/usa-process-list';
+@import '../../global/formation_overrides';
+
+
+:host([uswds]) {
+  counter-reset: usa-numbered-list;
+}
+
+:host([uswds]) ::slotted(va-process-list-item[checked='true']):before {
+  color: var(--color-green) !important;
+  border-color: var(--color-green) !important;
+  content: url(./green-check.svg) !important;  
+}
+
+:host([uswds]) ::slotted(va-process-list-item[active='true']):before {
+  color: var(--color-primary) !important;
+  border-color: var(--color-primary) !important;
+}
+
+:host([uswds]) ::slotted(va-process-list-item[pending='true']):before {
+  color: var(--color-gray-light) !important;
+  border-color: var(--color-gray-light) !important;
+}
+
+:host([uswds]) ::slotted(va-process-list-item[pending='true']) {
```

Comment:
@ataker I saw this was resolved but I'm not sure if it has been. Do you need `:host([uswds])` for these styles since `va-process-list-item` is only ever used with uswds?

Reply:
Oh good call!